src/libraries/System.IO.FileSystem/tests/FileInfo/GetSetTimes.cs

```diff
@@ -194,9 +194,8 @@ public void DeleteAfterEnumerate_TimesStillSet()
 
             DateTime afterTime = DateTime.UtcNow.AddSeconds(1);
 
-            // Deleting doesn't change any info state
             info.Delete();
-            ValidateSetTimes(info, beforeTime, afterTime);
+            Assert.True(info.LastAccessTimeUtc > afterTime);
```

Comment:
It'd be good to use Assert.InRange.  If the assert fails, we'll then get meaningful informationa about why.

Also, what about the other times?  ValidateSetTimes can't be used to validate those?

Reply:
After some debugging I actually think we should bin this test. On my local machine the invalidated date time ends up being `01/01/1601` (W10), I could change it so the time isn't in the series, or is simple different than both but I think the over-all goal of the test has been invalidated.

Reply:
Ah, we deleted the file. I think we have a test somewhere that validates the time we return for a non-existent file. Ideally we'd do the same check here. Otherwise, we can just delete the test.

Reply:
@stephentoub after some digging I actually think something is incorrectly happening here. https://github.com/dotnet/runtime/blob/master/src/libraries/System.IO.FileSystem/src/System/IO/FileSystemInfo.Windows.cs#L111 the underlying OS specific filesysteminfo is checking if it's been initialized, and stepping through it's set to 0, vs -1 so I'm investigating that

Reply:
Redacting my statement, I still think we should delete the test

Reply:
Why wouldn't we want to validate the times behave as they would for a nonexistent file?

Reply:
That makes sense to me too, but should be assert that it isn't in the range of those two times? Or just that the file has been defaulted to the system default

Reply:
Presumably we should check for the same values as in the earlier tests in this file.
https://github.com/Jlalond/runtime/blob/f204026ab55bbeae1c62252ba67bf61fdeefda5b/src/libraries/System.IO.FileSystem/tests/Base/InfoGetSetTimes.cs#L24

Reply:
Should've noticed that, eek. Implemented!